Update ReadMe, Site docs and fix broken links (#2177)
projectnessie/nessie: site/docs/features/management.md

spark session and a nessie owned `Catalog`. The `result` object above returns the number of files the action tried to delete and the number that failed.

!!! note

>     You can follow along an interactive demo in a [Jupyter Notebook via Google Colab](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/notebooks/nessie-iceberg-demo-nba.ipynb)

Reviewer: This should point to the Binder notebook, we no longer support Google Colab.

Author: yeah, below this line it has binder link as well. I shall remove colab link then.

Author: @snazy :  what about links to this (nessie-iceberg-spark-setup.ipynb) notebook ? I saw that recently during clean up this document is deleted. So, I will remove entire text talking about this document ?

Reviewer: No, don't delete it. Just replace the link with
```suggestion
    You can follow along an interactive demo in [this Notebook via Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-iceberg-demo-nba.ipynb)
```

Author: ok. I have changed as per your suggestion

## Compaction

!!! info

>     This service is currently in progress and is not yet included in a released version of Nessie.

Author: yes. I will look up all the doc once and handle it.

>     This service is currently in progress and is not yet included in a released version of Nessie.

Author: handled. only this place was pending
